app/src/test/kotlin/compiler/semantic_analysis/ArgumentResolverTest.kt

```diff
@@ -221,7 +221,9 @@ internal class ArgumentResolverTest {
         val nameResolution = referenceMapOf<Any, NamedNode>()
         val diagnostics = CompilerDiagnostics()
 
-        ArgumentResolver.calculateArgumentToParameterResolution(program, nameResolution, diagnostics)
+        try {
+            ArgumentResolver.calculateArgumentToParameterResolution(program, nameResolution, diagnostics)
+        } catch (_: ArgumentResolver.ResolutionFailed) { }
```

Comment:
If `ArgumentResolver` is expected to throw an exception, we should verify that the exception is actually thrown. The same comment goes to another test files.

Reply:
Done.